src/components/EntityInfo.vue

```diff
@@ -160,39 +174,23 @@
             </template>
 
             <!-- Email -->
-            <template v-if="businessId">
+            <template v-if="!!businessId">
               <dt class="mr-2">Email:</dt>
-              <dd
-                id="entity-business-email"
-                :class="isAllowed(AllowableActions.EDIT_BUSINESS_PROFILE) ? 'cursor-pointer' : 'pointer-events-none'"
```

Comment:
This is always allowed so I replaced the bound class with a static class.

Also below for phone.